web/templates/poll_modal_option.hbs

```diff
@@ -0,0 +1,8 @@
+<li class='option-row'>
+    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
+    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
```

Comment:
Rather than these doubled-up icons, which don't look quite right to me, we should see if @terpimost has a similar drag-handler icon that would work here.

Reply:
Have changed to using the [suggested single icon](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Modal.20to.20create.20polls/near/1641737), and here's how it looks now:
![image](https://github.com/zulip/zulip/assets/68962290/715d8c98-7be3-41f2-9e79-3f5eb6cf72ed)